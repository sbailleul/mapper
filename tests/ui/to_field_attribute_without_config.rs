use mapper_impl::Mapper;

#[derive(Mapper)]
#[to(Person)]
pub struct User{
    #[to(Person)]
    pub name: String
}

pub struct Person{
    pub name: String
}

fn main(){}