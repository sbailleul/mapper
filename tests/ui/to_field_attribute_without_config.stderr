error: To field attribute should contains configuration
 --> tests/ui/to_field_attribute_without_config.rs:6:10
  |
6 |     #[to(Person)]
  |          ^^^^^^
